tapdb/sqlc/migrations/000015_rfq.up.sql

```diff
@@ -0,0 +1,9 @@
+-- asset_sell_offers is a table that stores the set of all asset sell offers
+-- that are applicable to the RFQ negotiation process.
```

Comment:
Repeating the framing presented above:
 * When we're the first hop in the transfer, we're buying the asset and selling BTC. 
 * When we're the last hop in the transfer, we're selling the asset and buying BTC.